### PR TITLE
fix(schedule): don't update branch outside of schedule

### PR DESCRIPTION
**Pull request by meganbailey**
:tickets: TICKET-000 

## Changes
Set the [updateNotScheduled](https://docs.renovatebot.com/configuration-options/#updatenotscheduled) flag to `false` (it's `true` by default). This should lower our CI costs so that renovate doesn't update the PRs throughout the day and run CI every time something is merged.

### CODE REVIEW

File: library.json
Comment (prescottprue) [inline library.json:20]: I'm wondering if we can make a package rule in the base that would make it so that we wouldn't have to apply this to each one. That can come later though since it is kinda nice to see which rules it is applied with while we are testing

**Comment by meganbailey** [comment]
Sounds like @mathieudi has a fix already, so no need for this

**Comment by prescottprue** [comment]
@meganbailey I think it actually smart to go ahead and put this in place until devops has merged the work they are describing (which is on hold shortly).

@mathieudi and @skydiator we can remove ^ when we are turning off Renovate managing rebaseing/merge queue stuff

**Review by prescottprue — APPROVED** [review]

File: library.json
Comment (a88zach) [reply library.json:20]: @prescottprue @meganbailey I'll be interested to see if this works inside packageRules since it's not defined in the schema here:  https://docs.renovatebot.com/renovate-schema.json

Looks like this is applied at the root and can be moved to default.json

**Pull request merged**
